x-pack/plugins/monitoring/public/application/pages/page_template.tsx

```diff
@@ -5,16 +5,91 @@
  * 2.0.
  */
 
+import { EuiFlexGroup, EuiFlexItem, EuiTab, EuiTabs, EuiTitle } from '@elastic/eui';
 import React from 'react';
 import { useTitle } from '../hooks/use_title';
 
+export interface TabMenuItem {
+  id: string;
+  label: string;
+  description: string;
+  disabled: boolean;
+  onClick: () => void;
+  testSubj: string;
+}
 interface PageTemplateProps {
   title: string;
+  pageTitle?: string;
   children: React.ReactNode;
+  tabs?: TabMenuItem[];
 }
 
-export const PageTemplate = ({ title, children }: PageTemplateProps) => {
+export const PageTemplate = ({ title, pageTitle, tabs, children }: PageTemplateProps) => {
   useTitle('', title);
 
-  return <div>{children}</div>;
+  return (
+    <div className="app-container">
+      <EuiFlexGroup gutterSize="l" justifyContent="spaceBetween" responsive>
+        <EuiFlexItem>
+          <EuiFlexGroup
+            gutterSize="none"
+            justifyContent="spaceEvenly"
+            direction="column"
+            responsive
+          >
+            <EuiFlexItem>
+              <div id="setupModeNav">
+                <div
+                  className="monSetupModeEnterButton__buttonWrapper"
+                  data-test-subj="monitoringSetupModeBtn"
+                >
+                  <button
+                    className="euiButton euiButton--primary euiButton--small"
+                    type="button"
+                    style={{ border: '2px solid red' }}
```

Comment:
Could maybe take this out. It was an "idea" to help point out what's there but broken, but there's probably a bunch of stuff in that category that isn't called out in red. So might just be confusing as-is.

Reply:
I just removed it, keeping a JS comment to mark where the setup button and the timepicker should go.

Reply:
Perfect!